Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -70,8 +70,14 @@ ProjectPamphletViewModelOutputs {
     let refTag = self.refTagProperty.signal
       .map { $0.map(cleanUp(refTag:)) }
 
-    let liveStreamEventsFetch = project
-      .take(first: 1)
+    let projectWhenViewAppears = project.takeWhen(
+      self.viewWillAppearAnimated.signal
+    )
+
+    let liveStreamEventsFetch = Signal.merge(
+      project.take(first: 1),
+      projectWhenViewAppears
+      )
```

Comment:
of yer right: https://github.com/kickstarter/Kickstarter-ReactiveExtensions/blob/master/ReactiveExtensionsTests/operators/TakeWhenTests.swift#L130-L134

when the source completes, the whole thing completes. i always forget about that. i kinda think the correct behavior is to not complete, but don't wanna open that can of worms now.

you could still inline with this:

```swift
let liveStreamEventsFetch = Signal.combineLatest(
  project.take(first: 1)
  self.viewWillAppearAnimated.signal
  )
  .switchMap { ...
```